netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
```

Comment:
[it is so](https://github.com/firehol/netdata/pull/186/files/7154c1bad0b0aa18eddaa5b917104f108561c169#r58607212)

the `--with` and `--without` are optional if you wish to build with feature that is not enabled by default. or build without the feature which is enabled by default

you can read it this way:
- `%bcond_with foo` declares commandline option `--with foo`
- `%bcond_without foo` declares commandline option `--without foo`